tests/ha/ha_cluster_init.pm

```diff
@@ -22,13 +22,14 @@ sub run {
     my $bootstrap_log = '/var/log/ha-cluster-bootstrap.log';
     my $corosync_conf = '/etc/corosync/corosync.conf';
     my $sbd_device    = get_lun;
+    my $unicast_opt   = get_var("HA_UNICAST") ? '-u' : '';
```

Comment:
just for the record, same thing looks better and more widely used in our code base :
`my $unicast_opt   = get_var("HA_UNICAST") || '-u';`